src/sampleManifest.tsx

```diff
@@ -89,6 +90,7 @@ export const sampleManifest: SampleSpecGroup[] = [{
     getMarkerPinSpec(),
     getMultiViewportSpec(),
     getPropertyFormattingSpec(),
+    getHtmlMarkerSpec(),
```

Comment:
Any reason you put it here? This list is mostly alphabetical; although this one may be better off near MarkerPin

Reply:
good point... originally I had named it "react markers sample" which is why it was after `P`. I'll move it to MarkerPin.